### DISCUSSION TITLE
how to build android app for arm-v7a

### DISCUSSION THREAD

**Discussion by rogerwangzy** [rustdesk/rustdesk#690] [Q&A]
how to build android app for arm-v7a

**Comment by rustdesk** [comment]
Actually, we did not know either, never tried.

**Comment by Heap-Hop** [comment]
https://github.com/rustdesk/rustdesk/blob/master/flutter/build_android_deps.sh

https://github.com/rustdesk/rustdesk/blob/master/flutter/build_android.sh

#611
